src/controllers/User.js

```diff
@@ -0,0 +1,76 @@
+const User = require('../models/User');
+const jwt = require('jsonwebtoken');
+const { StatusCodes } = require('http-status-codes');
+const bcrypt = require('bcryptjs');
+const cookie = require('cookie');
+
+const UserController = async (req, res) => {
```

Comment:
[Naming]
This may have just been by accident. This handler function has been named UserController. Its not really the UserController, its just one function in the UserController. Looks like the getById function.

Reply:
I probably misunderstand you in a previous review, therefore I renamed the handler function back to getById.